<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор многочленов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
color: white;
background-color: #383838;
border: 2px solid #4a4a4a;
border-radius: 10px;
}

QPushButton:hover {
	background-color: #646464;
    border: 2px solid #828282;
}

QPushButton:pressed {
    background-color: #828282;
    border: 4px solid #AAAAAA;
}

QMainWindow, QWidget {
background-color: #2a2a2a;
}

QPlainTextEdit, QLineEdit, QTextEdit {
background-color: #242424;
color: white;
border: 2px solid #1b1b1b;
border-radius: 8px;
selection-background-color: gray;
padding: 10px;
}

QPlainTextEdit:focus, QLineEdit:focus, QTextEdit:focus {
    background-color: #1b1b1b;
}

QLabel {
	color: white;
    font-family: &quot;bahnschrift semibold&quot;, &quot;segoe ui&quot;;
    font-size: 20px;
}



QScrollBar:vertical {   
     border: 1px solid #191919; 
     background:white; 
     width:18px; 
     margin: 0px 0px 0px 0px; 
 border-radius: 10px;
 } 

 QScrollBar::handle:vertical { 
     background: #323232;
     min-height: 0px; 
	 border-radius: 10px;
 } 

 QScrollBar::handle:pressed {
	background-color: gray;
}

 QScrollBar::add-line:vertical { 
     height: 0px; 
 border-radius: 10px;
    background: #323232;
     subcontrol-position: bottom; 
     subcontrol-origin: margin; 
 } 

 QScrollBar::sub-line:vertical { 
     border-radius: 10px;
    background: #323232;
     height: 0 px; 
     subcontrol-position: top; 
     subcontrol-origin: margin; 
 } 

QScrollBar::sub-page:vertical, QScrollBar::add-page:vertical{
	background-color: black;
 border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="polynomsPlanTextEdit">
        <property name="font">
         <font>
          <family>Cascadia Mono</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: white;
margin: 10px;
margin-left: -6px;</string>
          </property>
          <property name="text">
           <string>Полученные многочлены</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>bahnschrift semibold</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>x = </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="donePolynomsPlanTextEdit">
        <property name="font">
         <font>
          <family>Cascadia Mono</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="inputExpressionTextEdit">
        <property name="font">
         <font>
          <family>Cascadia Mono</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="exEqualsLineEdit">
          <property name="font">
           <font>
            <family>Cascadia Mono</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
padding: 5px</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearPushButton">
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding: 10px;</string>
          </property>
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="solveEquationPushButton">
          <property name="font">
           <font>
            <family>Bahnschrift SemiBold</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string extracomment="Получить значение выражения"/>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:10px;
padding-left: 20px;
padding-right: 20px;</string>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="resultTextEdit">
        <property name="font">
         <font>
          <family>Cascadia Mono</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
